settings.gradle

```diff
@@ -46,4 +46,5 @@ include 'data-prepper-logstash-configuration'
 include 'e2e-test'
 include 'e2e-test:trace'
 include 'e2e-test:log'
+include 'data-prepper-test-common'
```

Comment:
It does not appear this project needs the test library. If that is so, please remove this from the PR.

Reply:
This is the settings.gradle file of `data-prepper` project. We need this to access matcher in other projetcs using
`testImplementation project(':data-prepper-test-common')`

Reply:
Sorry, my mistake here. I misread the file diff for the file name.

Reply:
Even I was confused after seeing the `data-prepper-logstash-configuration` on top